fs.go

```diff
@@ -0,0 +1,33 @@
+package gofs
+
+import "io"
+
+// ReaderFS is the interface that groups the basic Read file system methods.
```

Comment:
very sane interfaces =) I wonder when operating systems will deliver APIs like that...

Reply:
a man can dream :D